content/en/docs/concepts/overview/components.md

```diff
@@ -23,13 +23,13 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Master Components
+## Control Plane Components
```

Comment:
I think this is OK to change because even if fragment identifiers break, it's still the start of the page.
:+1: